Docs: Add custom permissions report docs (#30226)
grafana/grafana: docs/sources/enterprise/license-restrictions.md

License URL is the root URL of your Grafana instance. The license will not work on an instance of Grafana with a different root URL.

## Dashboard and folder permissions report

> This CSV report helps to identify users, teams and roles who have been granted "Admin" or "Edit" permissions at the dashboard or folder level.

Reviewer: ```suggestion
This CSV report helps to identify users, teams, and that have been granted `Admin` or `Edit` permissions at the dashboard or folder level.
```

> To download the report (in CSV format) go to **Server Admin** menu, select **Licensing**, scroll down to the bottom of the page and click on **Download report** button.

Reviewer: ```suggestion
To download the report (in CSV format) go to the **Server Admin** > **Licensing**. From the bottom of the page, select **Download report**.
```
